tests/test_iana_to_rflx.py

```diff
@@ -0,0 +1,250 @@
+import filecmp
```

Comment:
The unit tests have been restructured in 5d4de235. I chose to hold on to the separate files, as they now contain full specifications that in turn contain the tested types instead of only the tested types. Please have a look at it. If you still think that one big file would be better, I'll change that.